lib/engines/content_block_manager/app/views/content_block_manager/content_block/documents/new.html.erb

```diff
@@ -6,7 +6,7 @@
       href: content_block_manager.content_block_manager_content_block_documents_path,
     } %>
   <% end %>
-  <% content_for :page_title, "Select a block type" %>
```

Comment:
I've asked Chanelle if the error message needs to change as well https://gds.slack.com/archives/C062Z1FA2ES/p1733227350576849 
if we aren't sure atm can always be done in future PR